recipes/default.rb

```diff
@@ -15,51 +15,58 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-include_recipe 'osl-repos::centos'
-include_recipe 'osl-repos::epel'
-include_recipe 'osl-mysql::client'
-include_recipe 'base::python'
 
-# WARNING!
-# If this gets updated, all NodeJS apps running will need to have their
-# node_modules directories completely removed and `npm install` run again to
-# update the modules to match the new Node version's ABI.
-node.override['nodejs']['repo'] = 'https://rpm.nodesource.com/pub_6.x/el/$releasever/$basearch'
+if node['platform_version'].to_i <= 7
 
-# rvm package depends
-package %w(
-  automake
-  ImageMagick-devel
-  libffi-devel
-  libtool
-  libyaml-devel
-  openssl-devel
-  postgresql-devel
-  readline-devel
-  sqlite-devel
-  zlib-devel
-)
+  include_recipe 'osl-repos::centos'
+  include_recipe 'osl-repos::epel'
+  include_recipe 'osl-mysql::client'
+  include_recipe 'base::python'
 
-package 'osl-app packages' do
-  package_name osl_app_packages
-end
-package 'python-psycopg2' do
-  package_name 'python2-psycopg2' if node['platform_version'].to_i >= 8
-end
+  # WARNING!
+  # If this gets updated, all NodeJS apps running will need to have their
+  # node_modules directories completely removed and `npm install` run again to
+  # update the modules to match the new Node version's ABI.
+  node.override['nodejs']['repo'] = 'https://rpm.nodesource.com/pub_6.x/el/$releasever/$basearch'
 
-# Keep systemd services private from non-root users
-directory '/etc/systemd/system' do
-  mode '750'
-end
+  # rvm package depends
+  package %w(
+    automake
+    ImageMagick-devel
+    libffi-devel
+    libtool
+    libyaml-devel
+    openssl-devel
+    postgresql-devel
+    readline-devel
+    sqlite-devel
+    zlib-devel
+  )
+
+  package 'osl-app packages' do
+    package_name osl_app_packages
+  end
+
+  package 'python-psycopg2' do
+    package_name 'python3-psycopg2' if node['platform_version'].to_i >= 8
+  end
 
-# rewind the sudoers template to support sudoers_d
-temp = resources(template: '/etc/sudoers')
-temp.variables['include_sudoers_d'] = true
+  # Keep systemd services private from non-root users
+  directory '/etc/systemd/system' do
+    mode '750'
+  end
 
-build_essential 'install tools'
+  # rewind the sudoers template to support sudoers_d
+  temp = resources(template: '/etc/sudoers')
+  temp.variables['include_sudoers_d'] = true
+
+  build_essential 'install tools'
+
+end
 
 include_recipe 'git'
-include_recipe 'osl-nodejs'
+
+include_recipe 'osl-nodejs' if node['platform_version'].to_i <= 7
```

Comment:
If this can just be moved into the CentOS 7 if block above, let's do that please. I don't recall if this requires git or not. If it does, then put that before the if block.
```suggestion
include_recipe 'osl-nodejs' if node['platform_version'].to_i < 8
```